imap_processing/tests/spice/test_time.py

```diff
@@ -50,3 +56,47 @@ def test_sct2e_wrapper(sclk_ticks):
         assert isinstance(et, float)
     else:
         assert len(et) == len(sclk_ticks)
+
+
+def test_str_to_et(furnish_time_kernels):
+    """Test coverage for string to et conversion function."""
+    utc = "2017-07-14T19:46:00"
+    # Test single value input
+    expected_et = 553333629.1837274
+    actual_et = str_to_et(utc)
+    assert expected_et == actual_et
+
+    # Test list input
+    list_of_utc = [
+        "2017-08-14T19:46:00.000",
+        "2017-09-14T19:46:00.000",
+        "2017-10-14T19:46:00.000",
+    ]
+    # TODO: would this ^ be more of a realistic expectation as an array rather than a
+    # list?
```

Comment:
Just a general question for people reviewing here. I'm not sure what type of input is more likely when an iterable needs to be passed into this function - an array or a list?